src/test/java/uk/gov/justice/hmiprobation/casesampler/controllers/SampleControllerIntegrationTest.kt

```diff
@@ -53,11 +54,32 @@ class SampleControllerIntegrationTest(
             LocalDate.of(2019, 2, 26),
             "Westeros", "Winterfell", "Stark", "ZN", "XA", "Ned")
 
+    val exampleRequest = """
+                       [{
+                          "familyName" : "BK",
+                          "firstName" : "BK",
+                          "dob" : "13/11/1981",
+                          "gender" : "M",
+                          "sentenceType" : "POST_CUSTODY",
+                          "crn" : "R11111",
+                          "pnc" : "2000/0123456Q",
+                          "roshClassification" : "MEDIUM",
+                          "startDate" : "09/11/2018",
+                          "endDate" : "26/02/2019",
+                          "cluster" : "Westeros",
+                          "ldu" : "Winterfell",
+                          "team" : "Stark",
+                          "responsibleOfficer" : "ZN",
+                          "manager" : "XA",
+                          "officer" : "Ned"
+                        }]
+                    """".trimIndent()
+
     @BeforeEach
     fun setUp() {
-        Mockito.reset(caseListService)
+        Mockito.reset(caseSamplingService)
 
-        `when`(caseListService.process(any(), any(), any())).thenReturn(PrimaryCaseSampleProvisional(
```

Comment:
if using mockitokotlin2 then better to do a `whenever`

Reply:
not currently using that, I was using "com.ninja-squad:springmockk:2.0.1" but couldn't get it working with spring tests. It used to just ignore setting up stubbing on `mockBean`s